applications/crossbar/src/modules/cb_att_handlers_errors.erl

```diff
@@ -0,0 +1,133 @@
+-spec validate(cb_context:context()) -> cb_context:context().
+validate(Context) ->
+    validate_requests(Context, cb_context:req_verb(Context)).
+
+-spec validate(cb_context:context(), path_token()) -> cb_context:context().
+validate(Context, Id) ->
+    validate_request(Context, Id, cb_context:req_verb(Context)).
+
+-spec validate(cb_context:context(), path_token(), path_token()) -> cb_context:context().
+validate(Context, Resource, Id) ->
+    validate_request(Context, Resource, Id, cb_context:req_verb(Context)).
+
+-spec validate_requests(cb_context:context(), http_method()) -> cb_context:context().
+validate_requests(Context, ?HTTP_GET) ->
+    summary(Context).
+
+-spec validate_request(cb_context:context(), path_token(), http_method()) -> cb_context:context().
+validate_request(Context, Id, ?HTTP_GET) ->
+    read(Id, Context).
+
+-spec validate_request(cb_context:context(), path_token(), path_token(), http_method()) -> cb_context:context().
+validate_request(Context, Resource, Id, ?HTTP_GET) ->
+    read(Resource, Id, Context).
+
+%%%=============================================================================
+%%% Internal functions
+%%%=============================================================================
+
+%%------------------------------------------------------------------------------
+%% @doc
+%% @end
+%%------------------------------------------------------------------------------
+-spec read(kz_term:ne_binary(), cb_context:context()) -> cb_context:context().
+read(<<Year:4/binary, Month:2/binary, _/binary>> = Id, Context) ->
+    MODB = kazoo_modb:get_modb(cb_context:account_id(Context), Year, Month),
+    Context1 = cb_context:set_account_db(Context, MODB),
+    crossbar_doc:load(Id, Context1, ?TYPE_CHECK_OPTION(<<"attachment_handler_error">>)).
+
+-spec read(kz_term:ne_binary(), kz_term:ne_binary(), cb_context:context()) -> cb_context:context().
+read(<<"handler">>, HandlerId, Context) ->
+    Options = [{'mapper', fun normalize_modb_view_results/2}
+              ,{'range_keymap', [HandlerId]}
+              ],
+    crossbar_view:load_modb(Context, ?CB_LIST_BY_HANDLER, Options).
+
+%%------------------------------------------------------------------------------
+%% @doc
+%% @end
+%%------------------------------------------------------------------------------
+-spec summary(cb_context:context()) -> cb_context:context().
+summary(Context) ->
+    Options = [{'mapper', fun normalize_modb_view_results/2}],
+    crossbar_view:load_modb(Context, ?CB_LIST_ALL, Options).
+
+%%------------------------------------------------------------------------------
+%% @doc Normalizes the results of a view.
+%% @end
+%%------------------------------------------------------------------------------
+-spec normalize_modb_view_results(kz_json:object(), kz_json:objects()) -> kz_json:objects().
+normalize_modb_view_results(JObj, Acc) ->
+    [kz_doc:public_fields(kz_json:get_value(<<"value">>, JObj)) | Acc].
```

Comment:
you can use fun `crossbar_view/map_value_fun/0` as mapper.

Also for FYI you don't need to call `kz_doc:public_fields` here. it is already handled by crossbar_view or crossbar_doc